NitroxServer/GameLogic/Entities/Spawning/BatchEntitySpawner.cs

```diff
@@ -89,6 +89,18 @@ public List<Entity> LoadUnspawnedEntities(Int3 batchId)
                 Log.Info("Spawning " + entities.Count + " entities from " + spawnPoints.Count + " spawn points in batch " + batchId);
             }
 
+
+            for (int x = 0; x < entities.Count; x++)
```

Comment:
Was this for testing?  If not, let's break into a new function.  Also, it might be cleaner/faster to index in a map then loop through that.

Reply:
Let's do this one as well.